### DISCUSSION TITLE
datasets not collected

### DISCUSSION THREAD

**Discussion by gbeasse** [OpenLineage/OpenLineage#1184] [Q&A]
Hello,
I'm trying to test openlinage for airflow, interfaced with marquez.
When I consult the result in Marquez, I can only see the list of jobs run in my dag, but no datasets are found.  And no link between the different job.
I have simply updated my Dag to replace the imported dependances.
And I have added a environnement variable in airflow to specify the URL of marquez API. 
The versions I'm using are as follow : 
  python : 3.7
  airflow : 1.10.10 
  openlineage-airflow : 0.6.2

Do you have any idea why it does not send any informations about datasets ?

Regards,
Guillaume

**Comment by mobuchowski** [comment]
Hey, that's very old version of OpenLineage (and Airflow!) library. Could you try newer version?

**Reply by gbeasse** [reply]
Yes, I know, but unfortunately my client uses that old version of airflow :(  (He plans to migrate to airflow 2 next year)
I'm not even sure that 1.10.10 it's compatible with openlineage ^^

**Reply by mobuchowski** [reply]
It is, at least with 1.10.15 - that's the version we've tested with. 

Actually, it was compatible until ~20 days ago where we've removed the support, but 0.15.1 - latest released version should still have the support.

**Reply by gbeasse** [reply]
So according to you I should be able to collect datasets information, even if I'm running under airflow 1.10.10 ?
I haven't succeded in installing 0.15.1 with my current configuration (airflow 1.10.10).
That's why I've installed an older version, the highest version I have been able to install with no issue.

**Reply by mobuchowski** [reply]
There might be some significant differences between 1.10.10 and 1.10.15 - I'm not sure. 
I can take a look at the error/stack trace of the error of 0.15.1. 
Just no guarantees about actual support/fixing that bug, Airflow 1.10 is long deprecated.

**Reply by gbeasse** [reply]
Hello, please find attached my error when trying to install 0.15.1 using  "_pip3 install openlineage-airflow==0.15.1_"
I tried to install it from source as well and I got a similar error.
[install_openlineage.log](https://github.com/OpenLineage/OpenLineage/files/9886323/install_openlineage.log)
